### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by ACitronella**

### CODE REVIEW

File: lab-exercise-1/PhuriwatFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class PhuriwatFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (TreephobKho) [inline lab-exercise-1/PhuriwatFeature.java:3]: Please include this part.

Comment (Pakin-aw) [inline lab-exercise-1/PhuriwatFeature.java:3]: Nice comment.

File: lab-exercise-1/PhuriwatFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class PhuriwatFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/PhuriwatFeature.java:1]: good feature

File: lab-exercise-1/PhuriwatFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    public void feature() {
```
Comment (ACitronella) [inline lab-exercise-1/PhuriwatFeature.java:2]: *sigh*

File: lab-exercise-1/PhuriwatFeature.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+        // this is a new feature
+    }
```
Comment (sorrymaker1130) [inline lab-exercise-1/PhuriwatFeature.java:4]: ICT ICT ICT_T

**Comment by ACitronella** [comment]
Fix is ready

File: lab-exercise-1/PhuriwatFeature.java [hunk 5]
```diff
@@ -1,5 +1,7 @@
 public class PhuriwatFeature {
     public void feature() {
         // this is a new feature
+        int x = 0;
```
Comment (TreephobKho) [inline lab-exercise-1/PhuriwatFeature.java:4]: Thank you for the edit.

File: lab-exercise-1/PhuriwatFeature.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+public class PhuriwatFeature {
+    public void feature() {
+        // this is a new feature
+        int x = 0;
+        x = x + 1;
```
Comment (Pakin-aw) [inline lab-exercise-1/PhuriwatFeature.java:5]: Nice implementation.

File: lab-exercise-1/PhuriwatFeature.java [hunk 7]
```diff
@@ -0,0 +1,7 @@
+public class PhuriwatFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (RolinAnn) [inline lab-exercise-1/PhuriwatFeature.java:3]: Nice feature.

File: lab-exercise-1/PhuriwatFeature.java (hunk 6 above)
Comment (sorrymaker1130) [inline lab-exercise-1/PhuriwatFeature.java:5]: Add the part successful.

File: lab-exercise-1/PhuriwatFeature.java [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+        int x = 0;
```
Comment (RolinAnn) [inline lab-exercise-1/PhuriwatFeature.java:4]: Thank you for adding this part.

File: lab-exercise-1/PhuriwatFeature.java [hunk 9]
```diff
@@ -0,0 +1,7 @@
+public class PhuriwatFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/PhuriwatFeature.java:1]: good

**Pull request merged**